submissions/Shrai-dev/friends-app/app.js

```diff
@@ -0,0 +1,119 @@
+const CONTAINER = document.querySelector(".container");
+const SEARCH_BAR = document.querySelector(".searchBar");
+const SORTING_MENU = document.querySelector(".wrapper__filters");
+const SORT_GENDER = document.querySelector(".search__by-gender");
+const FRIENDS_NUMBER = 100;
+const URL = `https://randomuser.me/api/?results=${FRIENDS_NUMBER}`;
+let friends = [];
+
+const fetchPeople = (url) =>
```

Comment:
Your function will fetch "only" people. So you can have your URL and friends number here.